content/webapp/components/WorksRelatedContentCard/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { FunctionComponent } from 'react';
+
+import { convertIiifImageUri } from '@weco/common/utils/convert-image-uri';
+import LabelsList from '@weco/common/views/components/LabelsList';
+import WorkLink from '@weco/content/components/WorkLink';
+import { WorkBasic } from '@weco/content/services/wellcome/catalogue/types';
+
+import {
+  Card,
+  ImageWrapper,
+  LineClamp,
+  MetaContainer,
+  TextWrapper,
+  Title,
+} from './WorksRelatedContentCard.styles';
+
+type Props = {
+  work: WorkBasic;
+  resultIndex: number;
+};
+
+const WorksRelatedContentCard: FunctionComponent<Props> = ({
+  work,
+  resultIndex,
+}) => {
+  const {
+    productionDates,
+    title,
+    cardLabels,
+    thumbnail,
+    primaryContributorLabel,
+  } = work;
+  return (
+    <WorkLink
+      id={work.id}
+      resultPosition={resultIndex}
+      source={`works_search_result_${work.id}`}
```

Comment:
not sure this is right here, but `WorkLink` needed a `source` and this made it happy